backend/expense/src/main/java/org/egov/digit/expense/repository/querybuilder/BillQueryBuilder.java

```diff
@@ -153,7 +197,7 @@ private void addToPreparedStatement(List<Object> preparedStmtList, Collection<St
     }
 
     public String getSearchCountQueryString(BillSearchRequest billSearchRequest, List<Object> preparedStmtList) {
-        String query = getBillQuery(billSearchRequest, preparedStmtList,true);
+        String query = getBillQuery(billSearchRequest, preparedStmtList,true, false);
```

Comment:
Why is this value false here, where will be true?

Reply:
Count query is never called through validator, so it is kept true